app/models/user-registration.ts

```diff
@@ -2,17 +2,27 @@ import { attr } from '@ember-decorators/data';
 import { computed } from '@ember/object';
 import { buildValidations, validator } from 'ember-cp-validations';
 import DS from 'ember-data';
+import config from 'ember-get-config';
 
 const { Model } = DS;
 
+const { support: { supportEmail } } = config;
```

Comment:
Out of curiosity, would `const { supportEmail } = config.support;` have worked as well here?

Reply:
Yep yep, that's equivalent (and more readable 👍)

Reply:
Yes